docs/03-TheOrchestrator.md

```diff
@@ -0,0 +1,95 @@
+# Orchestrator
+
+The Orchestrator is a sidecar application to a Cosmos chain responsible for *orchestrating* transactions between Cosmos and Ethereum chains. In the Cosmos -> Ethereum direction, validators running the orchestrator with the Relayer enabled will query the [Gravity Module](https://github.com/peggyjv/gravity-bridge/tree/main/module/x/gravity) for pending transactions, and then send them to the gravity contract. In the Ethereum -> Cosmos direction, the Orchestrator acts as an oracle for the Cosmos chain to inform it of incoming transactions. This allows the coordination of appropriate locking/unlocking/minting/burning of tokens on either side of the bridge. The Orchestrator ensures that Gravity module state on the Cosmos chain is synchronized with the state of the gravity contract.
+
+## Quickstart
+
+The Orchestrator and Relayer can be started using the following command:
+
+```bash
+steward -c <config toml path> orchestrator start --ethereum-key <eth_key_name> --cosmos-key <cosmos key name>
+```
+
+If you do not wish to participate in Relaying Cosmos -> Ethereum transactions, you can start the orchestrator without the relayer by adding the `--orchestrator-only` flag:
```

Comment:
Given the above comment about relaying, we should probably flip the order of these and encourage people to run `--orchestrator-only` as the default for now.